query/src/sql/planner/binder/project.rs

```diff
@@ -79,7 +79,9 @@ impl Binder {
     ) -> Result<BindContext> {
         let mut output_context = BindContext::create();
         output_context.expression = input_context.expression.clone();
-
```

Comment:
The needed column bindings are added later, this may cause duplication.

Reply:
Yes, after applying your following suggestion, these codes are useless.